charts/sn-platform/values.yaml

```diff
@@ -2411,4 +2416,5 @@ istio:
     # default namespace for gateway is `istio-system`
     namespace: ""
     tls:
+      mode: SIMPLE
```

Comment:
Pls add comments for possible values of tls.mode, "SIMPLE", "STRICT"...

Reply:
Sure